superduperdb/db/base/artifact.py

```diff
@@ -206,15 +212,24 @@ def update(self, object: 'Component', metadata_info: t.Dict = {}):
 
         return self.replace(serialized, artifact_details)
 
-    def load(self, info, cache: t.Dict = {}):
+    def load(self, info, cache: t.Dict = {}, lazy: bool = False):
         """
         Recursively search through a tree for ArtifactDescs by an Artifact with
         that `file_id`.
 
         :param info: A tree made up of dicts and lists
         :param cache: A cache dictionary of Artifacts, keyed by file_id
         """
-        return self._load_artifacts(info, self.load_artifact, cache)
+
+        def no_load(**kwargs):
```

Comment:
check my above comment, 
we should return an EmptyArtifact like object which represents None like Artifact, so that __repr__ is not messed up?